core/languages/de.py

```diff
@@ -338,5 +338,11 @@
     "internal server error": "Interner Serverfehler",
     "not implemented": "Nicht implementiert",
     "bad gateway": "Falscher Gateway",
-    "service unavailable": "Dienst nicht verfügbar"
+    "service unavailable": "Dienst nicht verfügbar",
+
+    #Moduleconfig
+    "modulename":"Modulename",
+    "module helptext" : "Module Hilfetext",
```

Comment:
Isn't the German word for _module_  _Modul_?
```suggestion
    "modulename": "Modulname",
    "module helptext" : "Modul Hilfetext",
```